roles/falcon_installation/tasks/main.yml

```diff
@@ -1,21 +1,10 @@
 ---
 # tasks file for falcon_installation
```

Comment:
@redhatrises should this even be in here? If there is already a role for uninstallation, then we should abstract it out of the `falcon_installation` role. 

I would remove any variables and mentions of it in that role. If someone would like to remove/uninstall Falcon sensor - they can simply call the role. My 2c.